week-1-ts-ecosystem/w01e02-enter-the-world-of-typescript/src/shop/cart-item-sum.component.ts

```diff
@@ -0,0 +1,10 @@
+import { div } from '../framework/dom-creators';
+import { CartItemSum } from '../types';
+
+export function cartItemSum({ value, currency = 'PLN' }: CartItemSum): HTMLElement {
```

Comment:
Tak naprawdę zwracany jest typ `HTMLDivElement` więc robiąc `HTMLElement` - go zawęziłeś @michal-damiecki  :) Zawsze sobie zobacz wnioskowanie typów (najedź na to co zwraca `div` - chociaż Najpierw w tym układzie powinieneś poprawić to co jest we `framework`